mobile/src/navigation/course/course.navigation.tsx

```diff
@@ -42,13 +47,24 @@ const Course: FC = () => {
       screensByAuth,
       userPermissions,
     );
+    const hasInterview = Boolean(
+      interviewers.filter(({ interviewer }) => interviewer.id === user?.id)
+        .length,
+    );
+    const isShowMyMentorTabs = hasInterview && CourseScreenName.MY_MENTOR;
```

Comment:
This variable can be either boolean or string, should be only one type

Reply:
Can I use somthink like:  `const isShowMyMentorTabs = hasInterview ? CourseScreenName.MY_MENTOR : '' ` ?)

Reply:
No, looks not logical

Reply:
Variables that starts with is/has/should should be boolean

Reply:
Let's do something like this:
```typescript
const screenNameToFilter = useMemo(() => {
  if (isMentor && hasInterview) {
    return CourseScreenName.MY_MENTOR;
  }

 if (!isMentor) {
   return CourseScreenName. MY_STUDENTS;
 }